src/main/java/database/Storage.java

```diff
@@ -0,0 +1,16 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> assortment = new HashMap<>();
```

Comment:
It's a good practice to encapsulate the data in your classes. Currently, the `assortment` map is directly accessible and can be modified outside of the Storage class. This can lead to unpredictable results and bugs. You should make it private and provide public methods to manipulate it. Also, it's a good practice to use more specific names for your variables. The name `assortment` is not very descriptive. Maybe `fruitStorage` would be a better name?